src/meta/processors/jobMan/StatisJobExecutor.cpp

```diff
@@ -57,6 +57,56 @@ StatisJobExecutor::executeInternal(HostAddr&& address, std::vector<PartitionID>&
                                  std::move(parts), concurrency_, &(statisItem_[address]));
 }
 
+void showStatisItem(const cpp2::StatisItem& item, const std::string& msg) {
```

Comment:
What is this interface for?

Reply:
for debug ... and I think it maybe used one day. If you don't like it, I'm ok to remove those.